app/Actions/Place/GetPlaceTagsByCategory/GetPlaceTagsByCategoryAction.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Hedonist\Actions\Place\GetPlaceTagsByCategory;
+
+use Hedonist\Repositories\Place\PlaceCategoryTagCriteria;
+use Hedonist\Repositories\Place\PlaceCategoryTagRepository;
+
+class GetPlaceTagsByCategoryAction
+{
+    private $categoryTagsRepository;
+
+    public function __construct(PlaceCategoryTagRepository $repository)
+    {
+        $this->categoryTagsRepository = $repository;
+    }
+
+    public function execute(GetPlaceTagsByCategoryRequest $request): GetPlaceTagsByCategoryResponse
+    {
+        $tags = $this->categoryTagsRepository->findByCriteria(
+            new PlaceCategoryTagCriteria($request->getCategoryId())
```

Comment:
Easier probably to add categoryTagsRepository->findByCategory(int $categoryId) method and use `whereHas` inside

Reply:
Done.

Reply:
I think so too, but I thought that as we have `findByCriteria` method we should use it for more complicated requests.
I changed logic and add `findByCategory` method into `PlaceCategoryTagRepository` 'cause it's common repository for tags but `categoryTagsRepository` is for places tags in use, as I understand.